### DISCUSSION TITLE
Upgraded to Skip 1.1.3 getting FAILED skip export

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#245] [Q&A] [answered]
I just upgraded to skip 1.1.3  (along with MacOS 15 & Xcode 16)

Trying to deploy project on Play Store.
![Screenshot 2024-09-19 at 5 04 22 PM](https://github.com/user-attachments/assets/78f78266-b143-47a0-83ca-c1c3758a4c72)
 
That's a lot of logging with no idea what I'm looking for... Help

❯ wc /tmp/skip-export-2024-09-19T23:02:32Z.txt
    2401   68456 2837973 /tmp/skip-export-2024-09-19T23:02:32Z.txt
[skip-export-2024-09-19T23:02:32Z.txt](https://github.com/user-attachments/files/17067721/skip-export-2024-09-19T23.02.32Z.txt)

**Comment by aabewhite** [comment]
I haven't looked at the logs, but a common issue is a stale Package.resolved. You might try updating or deleting it

**Comment by davidakoontz** [comment, marked as answer]
I think my issue is I have an /Applications/Xcode 15.app  & Xcode 16.app  ( me and the Naming of things again!!)

The last line (yes I skipped ahead) has an Error that lead me to:
https://stackoverflow.com/questions/17980759/xcode-select-active-developer-directory-error

So after renameing the Xcode app & running a sudo command "repointing Xcode-select ...."

Skip Export is getting green check marks...

**Reply by davidakoontz** [reply]
skip export succeeded !!

[✓] Skip export ppwd to ./.build/skip-export (270.09s)
[✓] Skip 1.1.3 export succeeded in 270.1s

**Comment by davidakoontz** [comment]
Yeap my belief that names are mutable and that changing them should be done frequently until there is no desire to change them any more...  does NOT always serve me well.

Just DEPLOYED to Play Store again.
